kloppy/_providers/signality.py

```diff
@@ -0,0 +1,56 @@
+from contextlib import ExitStack
+from typing import Optional, Iterable, List
+
+from kloppy.domain import TrackingDataset
+from kloppy.infra.serializers.tracking.signality import (
+    SignalityDeserializer,
+    SignalityInputs,
+)
+from kloppy.io import FileLike, open_as_file, expand_inputs
+
+
+def load(
+    meta_data: FileLike,
+    raw_data_feeds: Iterable[FileLike],
+    venue_information: FileLike,
+    sample_rate: Optional[float] = None,
+    limit: Optional[int] = None,
+    coordinates: Optional[str] = None,
+) -> TrackingDataset:
+    """
+    Load and deserialize tracking data from multiple input files.
+
+    Args:
+        meta_data (FileLike): File-like object containing metadata information of the game.
```

Comment:
"A file-like object" is very vague. It is better to refer to something that the user is familiar with. For example, here I would write "json feed containing the metadata information of the game"